pypdf/_doc_common.py

```diff
@@ -998,16 +1004,13 @@ def _build_outline_item(self, node: DictionaryObject) -> Optional[Destination]:
     def pages(self) -> List[PageObject]:
         """
         Property that emulates a list of :class:`PageObject<pypdf._page.PageObject>`.
-        this property allows to get a page or a range of pages.
-
-        For PdfWriter Only:
-        Provides the capability to remove a page/range of page from the list
-        (using the del operator).
-        Note: only the page entry is removed. As the objects beneath can be used
-        elsewhere.
-        A solution to completely remove them - if they are not used anywhere -
-        is to write to a buffer/temporary file and to load it into a new PdfWriter
-        object afterwards.
+        This property allows to get a page or a range of pages.
+
+        For PdfWriter only: provides the capability to remove a page/range of
```

Comment:
* Please start with an uppercase letter after the colon.
* Please avoid two whitespace characters between words/sentences.
* Please use Sphinx-syntax to format the note accordingly.

Reply:
Will do. I will change the "Note: " to "Remember:" to avoid having a note within a note.